components/bl-pdfViewer-component/styles/index.less

```diff
@@ -0,0 +1,69 @@
+// you can use LESS variables from the current Theme and app extensions of the UI-Container
+@bl-customComponent-pdfViewer-themeColor: @themePrimary;
+@bl-customComponent-pdfViewer-themeTextColor: @appTextColor;
+
+@bl-customComponent-pdfViewer-input-background-color: if(@isLightTheme, darken(#fff, 10%), lighten(#000, 15%));
+@bl-customComponent-pdfViewer-button-icon-size: 16px;
+
+.bl-customComponent-pdfViewer {
+  color: @bl-customComponent-pdfViewer-themeTextColor;
+
+  .controls {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .controls-button {
+    padding: 0;
+    border: 0;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: 20px;
+    width: 60px;
+
+    fill: #fff;
+
+    border-radius: 4px;
+    background-color: @bl-customComponent-pdfViewer-themeColor;
+
+    cursor: pointer;
+
+    &.disabled {
```

Comment:
What about using pseudo class `:disabled` with buttons? In that case, we could remove duplication.

```
<button
        className={ cn('controls-button', { disabled: pageNumber === 1 }) }
        onClick={ () => setPageNumber(state => state - 1) }
        disabled={ pageNumber === 1 }>
        <PrevButtonIcon/>
      </button>
```

to

```
<button
        className={ cn('controls-button') }
        onClick={ () => setPageNumber(state => state - 1) }
        disabled={ pageNumber === 1 }>
        <PrevButtonIcon/>
      </button>
```